R/susie_wrapper.R

```diff
@@ -62,9 +62,11 @@ lbf_to_alpha <- function(lbf) {
 adjust_susie_weights <- function(twas_weights_results, keep_variants, allele_qc = TRUE,
                                  variable_name_obj = c("susie_results", context, "variant_names"),
                                  susie_obj = c("susie_results", context, "susie_result_trimmed"),
-                                 twas_weights_table = c("weights", context), combined_LD_variants, match_min_prop = 0.2) {
+                                 twas_weights_table = c("weights", context), combined_LD_variants, 
+                                 match_min_prop = 0.2, variant_name_flip=FALSE) {
```

Comment:
logically, if alleleQC is true then variant names should always be flipped. We should not need this parameter. In fact, in your new ordering of codes, the variant name flip must be TRUE. If allele is QC0-ed but variant name not flipped it is a bug. So please remove this paramter.

Reply:
Ok, i will have the default output variant names be flipped variant names

Reply:
it is fixed to output variant names that are flipped